quests/mysql.md

```diff
@@ -173,12 +173,14 @@ mentioned above does just this.
 <div class = "lvm-task-number"><p>Task 4:</p></div>
 
 Go back to your `site.pp` manifest and include the `mysql::server::account_security`
-class in the `learning.puppetlabs.vm` node. Remember, you don't need to pass any
+class in the `learning.puppetlabs.vm` node (after the class definition for ::mysql::server). 
```

Comment:
Change 'definition' to 'declaration' and let that note be part of the sentence instead of parenthetical.